ui/ad_position.js

```diff
@@ -91,7 +91,7 @@ shaka.ui.AdPosition = class extends shaka.ui.Element {
    */
   setPosition_() {
     const adsInAdPod = this.ad.getSequenceLength();
-    if (adsInAdPod > 1) {
+    if (adsInAdPod > 1 && this.ad.isLinear()) {
```

Comment:
It seems like the title of this PR is inaccurate.

The title says that we will HIDE the position when the ad is linear, but here it looks like you SHOW the position when the ad is linear.

Reply:
It was mistake, sorry! I updated the title